examples/complete/README.md

```diff
@@ -0,0 +1,10 @@
+# Complete Example
+
+An end-to-end basic example that will provision the following:
+
+- A new resource group if one is not passed in.
+- A new public SSH key if one is not passed in.
+- A new VPC with 3 subnets
+- A new placement group
+- A VSI in each subnet
+- A new Application Load Balancer
```

Comment:
A new Application Load Balancer to balance traffic between all virtual servers that are created by this example